js/scenes/Item.js

```diff
@@ -56,7 +57,9 @@ class Item extends Component {
         style={styles.image}
         source={{ uri: image }}
         resizeMode="cover"
-        ></Image>
+        >
+        <SocialShare/>
```

Comment:
Yeah you are right, this was just an example, was not planning to commit. Or I can commit showing how the example will work.